integration-tests/src/test/resources/features/AddressAPIHappyPath.feature

```diff
@@ -1,15 +1,16 @@
 Feature: Address API happy path test
+
   @address_api_happy
-  Scenario: Basic Address API journey
-    Given user has the user identity in the form of a signed JWT string
+  Scenario Outline: Basic Address API journey
+    Given user has the test-identity <testUserDataSheetRowNumber> in the form of a signed JWT string
```

Comment:
Is this the language that we're using over in the KBV API tests?

Reply:
Yes, pretty much, KBV API tests use an number for the row sheet.

Since we are using `Scenario Outline` and passing `Examples` for the tests to run, we need to declare that variable.

Reply:
Done.